ui/cypress/e2e/onboarding.test.ts

```diff
@@ -122,74 +130,88 @@ describe('Onboarding', () => {
       .should('be.enabled')
     cy.get('button:contains("Continue")').click()
 
-    //wait for new page to load
-    cy.location('pathname', {timeout: defTimeOut}).should(
-      'include',
-      'onboarding/2'
-    )
+    cy.wait('@orgSetup')
 
-    cy.getByTestID('notification-success').should($msg => {
-      expect($msg).to.contain('Initial user details have been successfully set')
-    })
+    //used in assertion below
 
-    cy.get('div[data-testid=notification-success] span.icon.checkmark').should(
-      'be.visible'
-    )
-    cy.get(
-      'div[data-testid=notification-success] button.notification-close'
-    ).should('be.visible')
+    cy.get('@orgSetup').then((xhr) => {
 
-    //check navbar
-    cy.get('div.wizard--progress-title.current').should($div => {
-      expect($div).to.contain('Complete')
-    })
-    cy.get('div.wizard--progress-title.checkmark').should($div => {
-      expect($div).to.contain('Welcome')
-      expect($div).to.contain('Initial User Setup')
-    })
+      let orgId: string = xhr.responseBody.org.id
 
-    cy.get('button.button-success')
-      .contains('Quick Start')
-      .should('be.visible')
-    cy.get('button.button-success')
-      .contains('Advanced')
-      .should('be.visible')
-    cy.get('button.button-success')
-      .contains('Configure Later')
-      .should('be.visible')
-
-    //advance to Quick Start
-    cy.get('button.button-success')
-      .contains('Quick Start')
-      .click()
-
-    cy.location('pathname', {timeout: defTimeOut}).should('include', '/me')
-
-    cy.getByTestID('notification-success').should($msg => {
-      expect($msg).to.contain(
-        'The InfluxDB Scraper has been configured for http://localhost:9999/metrics'
+      //wait for new page to load
+      cy.location('pathname').should(
+        'include',
+        'onboarding/2'
       )
-    })
 
-    cy.get('div[data-testid=notification-success] span.icon.checkmark').should(
-      'be.visible'
-    )
+      cy.getByTestID('notification-success').should($msg => {
+        expect($msg).to.contain('Initial user details have been successfully set')
+      })
 
-    cy.get(
-      'div[data-testid=notification-success] button.notification-close'
-    ).should('be.visible')
+      cy.get('div[data-testid=notification-success] span.icon.checkmark').should(
+        'be.visible'
+      )
+      cy.get(
+        'div[data-testid=notification-success] button.notification-close'
+      ).should('be.visible')
+
+      //check navbar
+      cy.get('div.wizard--progress-title.current').should($div => {
+        expect($div).to.contain('Complete')
+      })
+      cy.get('div.wizard--progress-title.checkmark').should($div => {
+        expect($div).to.contain('Welcome')
+        expect($div).to.contain('Initial User Setup')
+      })
+
+      cy.get('button.button-success')
+        .contains('Quick Start')
```

Comment:
we dont really care about the contents of these buttons.  just their functionality.

Reply:
This has been rewritten 
```javascript
     //advance to Quick Start
      cy.getByTestID('button')
        .eq(0)
        .contains('Quick Start')
        .click()
```
I've leveraged the existing data-testid attribute 'button', but with three matching elements on the page, I've continued with the `contains` assert, to ensure the test clicks the desired button.  If the number of buttons change or if they are rearranged, then the wrong button might get pressed.  This test checks onboarding to the page loaded after Quick Start.